### PR TITLE
Feature/sergeyro/avatar classname props

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/3875777031

### CODE REVIEW

File: src/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -55,6 +58,7 @@ const Avatar: React.FC<AvatarProps> & {
   id,
   type = AvatarType.TEXT,
   className,
+  circleClassName,
```
Comment (orrgottlieb) [inline src/components/Avatar/Avatar.tsx:61]: avatarContentWrapperClassName

**Review by orrgottlieb — APPROVED** [review]

File: src/components/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -26,7 +26,10 @@ export interface AvatarProps extends VibeComponentProps {
   withoutTooltip?: boolean;
   icon?: SubIcon;
   type?: AvatarType;
+  /** ClassName for a text type of AvatarContent */
```
Comment (hadasfa) [inline src/components/Avatar/Avatar.tsx:29]:
```suggestion
  /** The class name of the avatar content text*/
```

File: src/components/Avatar/Avatar.tsx [hunk 3]
```diff
@@ -26,7 +26,10 @@ export interface AvatarProps extends VibeComponentProps {
...
   textClassName?: string;
+  /** ClassName for a div-wrapper of AvatarContent */
```
Comment (hadasfa) [inline src/components/Avatar/Avatar.tsx:31]: Maybe can be confusing because avatar can be also square?
Maybe contentClassName?

Comment (SergeyRoyt) [reply src/components/Avatar/Avatar.tsx:31]: Orr proposed `avatarContentWrapperClassName`. Wdyt?

**Pull request merged**